### DISCUSSION TITLE
Tutorial - How to remove your camera stand with Photoshop

### DISCUSSION THREAD

**Discussion by byebyeworldg** [mot-commons/mot-commons#7] [Ideas]
https://vr360.work/workflow/adobephotoshop-panorama-image/

**Comment by byebyeworldg** [comment]
若干違和感がありますが、映像でも消せました。
一応やり方を書いておきます。
Adobe Premiere
下のレイヤーに映像、上のレイヤーに映像のスクショからカメラスタンド消した画像を配置。
エフェクトコントロールからマスクを選択し、スタンドの部分のみクリッピングして隠す。
エッジをぼかして滑らかにする。
カラーバランスなどで色を合わせる。（スクリーンなどの透明エフェクトも試しても良い。）

下の方なので、静止画で映像のちらつきがなくとも私は気になりませんでした。

**Reply by goh-works** [reply]
共有ありがとうございます．
THETAで記録した360°映像を，THETA以外の映像ソフトで編集した場合，360°音声を再現できるように書き出すために何か工夫がいると思います．（このTHETA映像は4chマイクで記録してて，規格に対応してるYoutubeに上げると３D音場再生ができます．映像内で音がどこからなってるかが分かる感じになります）
Premiereだったら，もしかしたらプラグイン等の簡潔なやり方があるかもしれません．

僕はDavinci resolveで編集したら360°音場が再現できなくなりました．この辺はちょっと面倒そうなので今回は保留にしました．

**Reply by goh-works** [reply]
THETAは動画向きでない気がするので別の機材になるのですが，360°で空間再現ができると強いので，映像も音場も撮影と演習方法を得られると良いですよね．Youtubeだとどちらも３D再生できるように規格が対応してるので発表するのにも困らないですし．まだやってないですけど，たぶんOculusとか繋いでYoutubeのVRボタン押したら，そのままVRになる気がします．
個人的に何か360°対応のカメラ買おうかと思ってるくらいです．

**Reply by byebyeworldg** [reply]
Gohさんがいただいた映像の音声4chでしたか？僕がいただいた2つの映像(各部屋映像１つ)のはどちらもステレオどころかモノ音声でした。１箇所からしか音がならないため不要と思われたのか、録画設定or出力の際に消してしまったのかわかりませんが、どちらにせよ私の分の今回の書き出しはメタデータを入れる必要がなくなりました。ちなみにPremiereにVRの設定があったので、そこらへんでできそうですね。

**Reply by byebyeworldg** [reply]
Premiereでの4ch編集と書き出し意外と簡単そうでした。
https://junoosuga.com/?p=668
音が1チャンネルしかなくとも、１箇所からしか音がでていない場合、音量を分けることで擬似的に位置を作れそうですね。

**Reply by goh-works** [reply]
ためしに未編集の元動画をVLCに入れてヘッドフォンで聞いてみてください．映像を動かすと音も動き，音にズームもできます．定位感がないような音だと分かりづらいかもしれません．
また下記の最後のYoutube用のTHETAソフトで映像を変換して，Youtubeに上げると，音場を再現できます．
https://theta360.com/ja/about/application/pc.html

動画の音データは１ｃｈで正しいです．
この空間音響のデータの規格は，複数チャンネルで録った音を１ｃｈに束ねて，それを生成時に展開するものです．
映像の音データは普通の１ｃｈに見えますが，それは空間音響が再現可能な規格のデータです．普通のモノラルデータと内部が違うのだと思います．
そしてそれを映像編集ソフトで普通に書き出すと，規格が壊れるようで，空間音響が再現できなくなりました．何かやり方があるのだと思います．

今回はTHETA Z1という上位機種で，たぶんこの機種だけが４ｃｈ録音での空間音響に対応してると思います（空間音響に対応した１ｃｈデータを吐き出します）．
他の機種は音データの扱いが違うかもしれません．

少し前にWikiで映像編集方法としてスマホアプリを勧めましたが，それは色調補正をしつつ空間音響データを維持してデータを書き出せるからでした（Youtubeにそのまま上げるだけで音場も再現できます）．
僕は個人的にTHETAは今後使わないと思うので，音の規格を調べていないままです．あまりお役に立てずすみません．

**Comment by byebyeworldg** [comment]
https://user-images.githubusercontent.com/82152582/114294450-b62e4b00-9ad9-11eb-98a2-85c18bb3aeb8.mp4

**Comment by byebyeworldg** [comment]
Gohさん
詳しくありがとうございます。
イヤホンでも試したのですが、違いが全く感じられませんでした。（YouTubeにアップされている参考動画でははっきりとわかりました。おもしろいですね。）
ただ、1chで正しいとのことなので、単純に私の音源にほぼ差異がないのでしょう。

ちなみに動画の表示スタート方角を変えたい場合、中心位置を帰れば良い様です。
Premeireであればエフェクト「ディストーション」にある「オフセット」で中心位置をずらすとできます。
写真はPSDを使うと「フィルタ」「その他」「スクロール」で横移動だけならできました。上下を変えたら亜空間みたいなのができました。

スタート時の表示倍率（zoomぐあい、どれくらいひくかなど）指定方法は見つけられませんでした。
もしかしてメタデータにないのかも。（？）

また、VRのSNSに関して、
Twitter（Twitterやっていないのですが）とInstagramは対応していない様です。意外でした。
Facebookはアップできました。YoutubeはGohさんがおっしゃっていた通りです。
